django/db/migrations/serializer.py

```diff
@@ -198,7 +198,10 @@ def serialize(self):
 class ModelFieldSerializer(DeconstructableSerializer):
     def serialize(self):
         attr_name, path, args, kwargs = self.value.deconstruct()
-        return self.serialize_deconstructed(path, args, kwargs)
+        try:
+            return self.serialize_deconstructed(path, args, kwargs)
+        except ValueError as e:
+            raise ValueError('Error during {} serializing: {}'.format(self.value, e)) from e
```

Comment:
Guidance is to try and avoid format. Do you think this could be more readable written as an f string or % formatting?

Reply:
Thanks, you're right. It's better to use an f string.